lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+extern crate wasm_bindgen;
+#[macro_use] extern crate slice_as_array;
```

Comment:
This is old-rust. You dont need to use it anymore. Just use normal `use` syntax instead: `use slice_as_array`.